### PR TITLE
[scripts] relax CSL ping timeout constraint

### PR DESCRIPTION
**Pull request by brianljt**

### CODE REVIEW

**Review by bukepo — APPROVED** [review]

**Review by Irving-cl — APPROVED** [review]
LGTM 👍

File: tests/scripts/thread-cert/v1_2_LowPower_7_2_01_ForwardTrackingSeries.py [hunk 1]
```diff
@@ -91,7 +91,7 @@ def test(self):
 
         # Step 3 - Verify connectivity by instructing each device to send an ICMPv6 Echo Request to the DUT
         self.assertTrue(self.nodes[SED_1].ping(leader_addr, timeout=POLL_PERIOD / 1000))
-        self.assertTrue(self.nodes[SSED_1].ping(leader_addr, timeout=2 * consts.CSL_DEFAULT_PERIOD_IN_SECOND))
+        self.assertTrue(self.nodes[SSED_1].ping(leader_addr))
```
Comment (simonlingoogle) [inline tests/scripts/thread-cert/v1_2_LowPower_7_2_01_ForwardTrackingSeries.py:94]: It does not remove timeout constraint, it sets the timeout constraint to default (5s).

Comment (brianljt) [reply tests/scripts/thread-cert/v1_2_LowPower_7_2_01_ForwardTrackingSeries.py:94]: Yes, it is just relaxing the constraint from 1 second to 5 seconds which makes the change of not being able to send out the CSL indirect Tx relatively smaller. Local test result shows no failure in 1000 iterations.

**Review by simonlingoogle — APPROVED** [review]
LGTM :+1:

**Review by jwhui — APPROVED** [review]

**Pull request merged**